src/broadcast/main.ts

```diff
@@ -1,27 +1,29 @@
 import { settingsManager } from "@settings/settingsManager";
 
-import { broadcastManager } from "./broadcastManager";
 import { ipc_refreshBroadcastList, ipc_startBroadcast, ipc_stopBroadcast, ipc_watchBroadcast } from "./ipc";
-import { spectateManager } from "./spectateManager";
+import { broadcastWorker, spectateWorker } from "./workerInterfaces";
 
 ipc_refreshBroadcastList.main!.handle(async ({ authToken }) => {
-  await spectateManager.connect(authToken);
-  await spectateManager.refreshBroadcastList();
+  const sWorker = await spectateWorker;
+  await sWorker.refreshBroadcastList(authToken);
   return { success: true };
 });
 
 ipc_watchBroadcast.main!.handle(async ({ broadcasterId }) => {
+  const sWorker = await spectateWorker;
   const folderPath = settingsManager.get().settings.spectateSlpPath;
-  spectateManager.watchBroadcast(broadcasterId, folderPath, true);
```

Comment:
We used to have singleton to be `true` when calling `spectateManager.watchBroadcast()`. This doesn't seem to be the case anymore. Is this intentional?

Reply:
```ts
  public watchBroadcast(broadcastId: string, targetPath: string, singleton?: true)
  ```
 the function signature forces true so I skipped it when i was rewriting. Do we want to put true anyways?

Reply:
I didn't realise it was true by default. It's fine to leave it as undefined.